### PR TITLE
Add minutes for TSC meeting on June 4th, 2020

### PR DESCRIPTION
**Pull request by mnonnenmacher**

### CODE REVIEW

File: minutes/2020-06-04.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Meeting Minutes for June 4, 2020
+
+## Follow-up on previous meetings
+
+### ort-geovernance repository
```
Comment (sschuberth) [inline minutes/2020-06-04.md:5]: "governance"

File: minutes/2020-06-04.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+Thomas: No, it's only available on Google Docs with restricted access. But the
+diff to the original charter can be seen
+[here]([https://github.com/oss-review-toolkit/ort-governance/commit/022d2ae5c6817547f778b3cc8f212c401b2786f1).
+
+## Agenda
+
+### Contributing file
+
+The contributing.md needs to be
```
Comment (sschuberth) [inline minutes/2020-06-04.md:22]: "`CONTRIBUTING.md`"

File: minutes/2020-06-04.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
+# Meeting Minutes for June 4, 2020
+
+## Follow-up on previous meetings
+
+### ort-geovernance repository
+
+Thomas: The repository was made public using CC-BY-4.0 as license and is now
+also used for the meeting minutes.
```
Comment (sschuberth) [inline minutes/2020-06-04.md:8]: Merge this to below's "ort-governance is now public" point?

File: minutes/2020-06-04.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  * Sebastian: There are problems contributing required changes to the Sonatype
+    client library, might require a fork of the library before we can integrate
+    it.
+
+### Read all files from config directory
+
+Sebastian: Work is started, but the layout of the directory needs to be
+discussed. No plans to work on this in the next weeks.
+
+Check if there is a GitHub issue for this and if not create one (Thomas).
```
Comment (sschuberth) [inline minutes/2020-06-04.md:75]: As I couldn't find an issue yet, I've created one at https://github.com/oss-review-toolkit/ort/issues/2756.

File: minutes/2020-06-04.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+### Various topics
+
+* Team agrees on adding the suggestion to the contributing.md to announce major
```
Comment (sschuberth) [inline minutes/2020-06-04.md:79]: "`CONTRIBUTING.md`"

Comment (mnonnenmacher) [reply minutes/2020-06-04.md]: Done.

File: minutes/2020-06-04.md (hunk 4 above)
Comment (mnonnenmacher) [reply minutes/2020-06-04.md]: Done.

File: minutes/2020-06-04.md (hunk 3 above)
Comment (mnonnenmacher) [reply minutes/2020-06-04.md]: Done.

File: minutes/2020-06-04.md (hunk 2 above)
Comment (mnonnenmacher) [reply minutes/2020-06-04.md]: Done.

File: minutes/2020-06-04.md (hunk 1 above)
Comment (mnonnenmacher) [reply minutes/2020-06-04.md]: Done.

**Pull request merged**